oioioi/forum/templatetags/display_reacted_by.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.utils.translation import gettext as _
+from oioioi.base.utils import get_user_display_name
+from django.conf import settings
+
+register = template.Library()
+
+@register.simple_tag
+def display_reacted_by(post, rtype):
+  # post needs to be queried with prefetch_reacted_by
+
+  output = ', '.join([
+    get_user_display_name(reaction.author)
+    for reaction in getattr(post, rtype.lower() + 'd_by')
```

Comment:
why `.lower()`, since it is an internal tag we can expect that someone will use it with proper rtype